app/helpers/judgements_helper.rb

```diff
@@ -0,0 +1,43 @@
+module JudgementsHelper
```

Comment:
I think this is far away from a nice solution, but it requires a larger refactor to improve. I think a proper view model should be created and should be used here and at evaluation too. Add 👍 if you agree and I will create a Trello card for this.